src/callbacks_step/amr_dg1d.jl

```diff
@@ -76,6 +76,39 @@ function refine!(u_ode::AbstractVector, adaptor, mesh::TreeMesh{1},
     return nothing
 end
 
+function refine!(u_ode::AbstractVector, adaptor, mesh::TreeMesh{1},
+                 equations, dg::DGSEM, cache, cache_parabolic,
+                 elements_to_refine)
+    refine!(u_ode, adaptor, mesh, equations, dg, cache, elements_to_refine)
```

Comment:
It would imho be good to add a comment in such cases to make it easier for the user to understand what's going on, e.g.
```suggestion
    # Call `refine!` for the hyperbolic part, which does the heavy lifting of
    # actually transferring the solution to the refined cells
    refine!(u_ode, adaptor, mesh, equations, dg, cache, elements_to_refine)
    
    # The remaining function only handles the necessary adaptation of the data structures
    # for the parabolic part of the semidiscretization
```

Something similar would of course be helpful for the `coarsen!` function.